catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStorePublisher.java

```diff
@@ -210,7 +210,7 @@ public void handleEvent(Event event) {
                 (RegistryStore) bundleContext.getService(((ServiceEvent) event.getProperty(
                         EventConstants.EVENT)).getServiceReference());
 
-        if (!previousAutoPush && registryStore.isAutoPush()) {
+        if (registryStore.isAutoPush()) {
```

Comment:
Previously, this was only publishing or un-publishing when auto push changed. The problem with this was that when the url was updated without changing auto push, it would not publish to the new url. This now attempts to publish on any change _if_ the url has not already been published to (the publish function checks published locations before attempting a new publish). It looks like the un-publish function also performs this check so I will take out `previousAutoPush` as it is not needed.